storage_service/locations/views.py

```diff
@@ -549,7 +552,15 @@ def pipeline_edit(request, uuid=None):
             return redirect("locations:pipeline_list")
     else:
         form = forms.PipelineForm(instance=pipeline, initial=initial)
-    return render(request, "locations/pipeline_form.html", locals())
```

Comment:
Thanks for making this `locals()` pattern explicit :bow:

Reply:
26 left :laughing: